src/main/groovy/com/github/lkishalmi/gradle/gatling/GatlingPlugin.groovy

```diff
@@ -83,7 +82,8 @@ class GatlingPlugin implements Plugin<Project> {
                     args "-m"
                     args "-bf", "${project.sourceSets.gatling.output.classesDir}"
                     args "-s", simu
-                    args gatlingExt.gatlingArgs(project)
+                    args "-df", "${project.sourceSets.gatling.output.resourcesDir}"
```

Comment:
Hello @lkishalmi 

During my work on the plugin and reading `Gatling` documentation I haven't seen any _technical_ reason for keeping bodies and feeders folders separately when invoking `Gatling` tool.

I assume this is a logical separation for the sake of better readability and project structure. From this point of view - we're still keeping it because bodies and feeders are located in different source folders, they are separate for customer and only mixed together in runtime, invisible for customer.

Moreover, I'm more inclined to use `resources` output folder instead of `resources` source folder because of 
* it is common way for many of `Gradle` plugins to take their data
* it is useful for cases when customer wants to preprocess or auto-generate feeder / bodies data by build script

So, I suggest for now to proceed with current approach, unless we receive customer feedback with complaints related to bodies/feeder data location.